plugins/services/src/js/utils/CreateServiceModalFormUtil.js

```diff
@@ -34,6 +34,119 @@ const CreateServiceModalFormUtil = {
   },
 
   /**
+   * Patch an object with an object
+   *
+   * This function:
+   * - Removes items whose patch value is `empty` (null, undefined, 0, or '')
```

Comment:
@nLight `0` on `5` keeps the same type, so it's going to be replaced with `0`. However because of your comment I found out a bug on the arrays. Pushing a fix 👍